pom.xml

```diff
@@ -28,7 +28,7 @@
     <parent>
         <groupId>org.jenkins-ci.plugins</groupId>
         <artifactId>plugin</artifactId>
-        <version>2.3</version>
+        <version>2.5</version>
```

Comment:
Should not. Different versions of the parent define different defaults for `jenkins.version`, `jenkins-test-harness.version`, and `java.level`, but we are defining all of them explicitly anyway.